WPMedia/fusion-news-theme-blocks
# Tmedia 708 sub headline subs

## Description

Use subheadline data not headline data

## Jira Ticket

- [TMEDIA-708](https://arcpublishing.atlassian.net/browse/TMEDIA-708)

## Acceptance Criteria
Actual Behavior
Subheadline is displaying the headline text.

Expected Behavior
Subheadline should be displaying the subheadline text.
## Test Steps

1. Checkout this branch `git checkout TMEDIA-708-sub-headline-subs`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/subscriptions-block`
3. Create page with an Offer on it
4. See data not duplicated 

<img width="1050" alt="Screen Shot 2022-03-21 at 14 49 20" src="https://user-images.githubusercontent.com/5950956/159352724-f0d4d84a-de59-4641-aafe-5999478c8e4a.png">
<img width="1226" alt="Screen Shot 2022-03-21 at 14 48 30" src="https://user-images.githubusercontent.com/5950956/159352740-854f104c-8d0f-4a4a-bf0d-781cf8e73927.png">


## Effect Of Changes

### Before

dup headlines
<img width="803" alt="Screen Shot 2022-03-21 at 14 53 32" src="https://user-images.githubusercontent.com/5950956/159353214-3ef43ada-224f-4d73-9154-18a9032cc921.png">

### After
<img width="1226" alt="Screen Shot 2022-03-21 at 14 48 30" src="https://user-images.githubusercontent.com/5950956/159353239-eb1a1193-94df-41b1-9a54-7fe47620f2e2.png">



## Dependencies or Side Effects

- none

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
